ruby/grape/config.ru

```diff
@@ -0,0 +1,25 @@
+Bundler.require :default
+
+module Bench
+  class BaseAPI < Grape::API
+    get '' do
+      ''
```

Comment:
Is the intent to return no content or a stirng? If it's the former, you need to return no content and a 204. Otherwise this returns a JSON empty string by default, invoking a JSON serializer.

Reply:
The purpose is to return an empty string. 

I do not understand what you are saying. CI is passing, it test that this endpoint return an empty string (but does not check the header though)

Reply:
Empty string in which content type? If you're returning `application/json`, it should return `""`, if it's `text/plain`, you can't return an empty string at all. If you want not to return content it should be a 204 without a body like you changed it to. What I am saying is that all these have radically different performance characteristics because serializing data, even an empty string, is not free.